extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/discovery/AstroDiscoveryService.java

```diff
@@ -66,10 +67,16 @@ protected void activate(Map<String, Object> configProperties) {
     }
 
     @Override
+    @Modified
     protected void modified(Map<String, Object> configProperties) {
         super.modified(configProperties);
     }
 
+    @Override
+    protected void deactivate() {
```

Comment:
As you aren't doing anything in here anymore, we do not have to keep this method as it does not carry any annotation anymore, right?

Reply:
hang on - `super` does! It's just the annotation which is missing here